api/Navigator.json

```
@@ -324,6 +324,42 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "User-Agent_reduction": {
+          "__compat": {
+            "description": "Provides limited information due to [User-agent](https://developer.mozilla.org/docs/Web/HTTP/Guides/User-agent_reduction) reduction.",
```

Here's my suggestion, but before applying, I'd like to get a second opinion by @ddbeck, who might have a better idea.

```suggestion
        "applies_UA_reduction": {
          "__compat": {
            "description": "Applies [User-agent reduction](https://developer.mozilla.org/docs/Web/HTTP/Guides/User-agent_reduction)",
```